### PR TITLE
Adding citation file

### PR DESCRIPTION
**Pull request by gvwilson**
Adding CITATION.cff file for this paper.

### CODE REVIEW

**Comment by ctb** [comment]
hi @gvwilson do you want us to add our ORCID's via PR, or just drop them here?

I did a PR b/c I don't have edit privileges - here. https://github.com/gvwilson/best-practices-scientific-computing/pull/1

Thanks for doing this!

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    orcid: ""
+  - family-names: "Waugh"
+    given-names: "Ben"
+    orcid: ""
+  - family-names: "White"
+    given-names: "Ethan P."
+    orcid: ""
+  - family-names: "Wilson"
+    given-names: "Paul"
+    orcid: ""
```
Comment (gonuke) [inline CITATION.cff]:
```suggestion
    orcid: "https://orcid.org/0000-0002-8555-4410"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  orcid: ""
+- family-names: "Waugh"
+  given-names: "Ben"
+  orcid: ""
+- family-names: "White"
+  given-names: "Ethan P."
+  orcid: ""
+- family-names: "Wilson"
+  given-names: "Paul"
+  orcid: ""
```
Comment (gonuke) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0002-8555-4410"
```

**Comment by gonuke** [comment]
Thanks for this update @gvwilson  - I've dropped my ORCID as a suggestion in code review

**Comment by beroe** [comment]
Hi Greg. Hope you are well. 
orcid: 0000-0001-9420-4482
Thanks, Steve

**Comment by ethanwhite** [comment]
Here's my ORCID:

0000-0001-6728-7745

**Comment by benwaugh** [comment]
My ORCID ID is 0000-0003-0872-8920. Thanks!

**Pull request closed without merging**